src/main/java/cz/cvut/kbss/analysis/dao/FaultTreeDao.java

```diff
@@ -45,10 +46,28 @@ public EntityDescriptor getEntityDescriptor(URI uri) {
         EntityType<FaultTree> ft = em.getMetamodel().entity(type);
         EntityType<FaultEvent> fe = em.getMetamodel().entity(FaultEvent.class);
         Attribute manifestingEvent = ft.getAttribute("manifestingEvent");
+        Attribute children = fe.getAttribute("children");
```

Comment:
@ledsoft  don't we have a generated metamodel in JOPA so we would not have to write the string "children"? If so I believe it is a good place to use it here.